cmd/kubectl-testkube/commands/scripts/start.go

```diff
@@ -47,7 +48,7 @@ func NewStartScriptCmd() *cobra.Command {
 				watchLogs(execution.Id, client)
 			}
 
-			execution, err = client.GetExecution("-", execution.Id)
+			execution, err = client.GetExecution("", execution.Id)
```

Comment:
and remove first parameter here as execution.Id is unique for us

Reply:
✅